packages/manager/apps/pci-public-ip/src/api/hooks/useSubnets.ts

```diff
@@ -1,14 +1,22 @@
 import { useQuery } from '@tanstack/react-query';
 import { getSubnets, getSubnetsUrl } from '@/api/data/subnets';
 
-export const getSubnetsQuery = (projectId: string, networkId: string) => ({
-  queryKey: [getSubnetsUrl(projectId, networkId)],
-  queryFn: () => getSubnets(projectId, networkId),
+const getSubnetsQuery = (
+  projectId: string,
+  region: string,
+  networkId: string,
+) => ({
+  queryKey: [getSubnetsUrl(projectId, region, networkId)],
+  queryFn: () => getSubnets(projectId, region, networkId),
 });
```

Comment:
```suggestion
const getSubnetsQuery = (
  projectId: string,
  region: string,
  networkId: string,
) => (queryOptions({
  queryKey: [getSubnetsUrl(projectId, region, networkId)],
  queryFn: () => getSubnets(projectId, region, networkId),
  enabled: !!projectId && !!region && !!networkId,
}));
```

To have the magic of typescript :) 
https://tanstack.com/query/latest/docs/framework/react/guides/query-options